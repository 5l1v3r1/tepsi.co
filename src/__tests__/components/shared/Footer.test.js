import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../../../components/shared/Footer';

it('renders correctly', () => {
  const wrapper = shallow(<Footer />);

  expect(wrapper).toMatchSnapshot();
});
