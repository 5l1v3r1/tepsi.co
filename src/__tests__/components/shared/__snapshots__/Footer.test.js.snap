// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="Footer"
>
  <nav
    className="Footer-menu"
  >
    <a
      className="Footer-menu__item"
      href="https://github.com/alpcanaydin/tepsi.co"
      rel="noopener noreferrer"
      target="_blank"
    >
      kaynağı görüntüle
    </a>
    <a
      className="Footer-menu__item"
      href="https://twitter.com/alpcanaydin"
      rel="noopener noreferrer"
      target="_blank"
    >
      @alpcanaydin
    </a>
    <a
      className="Footer-menu__item"
      href="https://twitter.com/pinquitte"
      rel="noopener noreferrer"
      target="_blank"
    >
      @pinquitte
    </a>
    <a
      className="Footer-menu__item"
      href="https://twitter.com/bigrandma"
      rel="noopener noreferrer"
      target="_blank"
    >
      @bigrandma
    </a>
    <a
      className="Footer-menu__item"
      href="https://twitter.com/batuhankrskl"
      rel="noopener noreferrer"
      target="_blank"
    >
      @batuhankrskl
    </a>
    <a
      className="Footer-menu__item"
      href="https://twitter.com/hsnaydd"
      rel="noopener noreferrer"
      target="_blank"
    >
      @hsnaydd
    </a>
  </nav>
</footer>
`;
